src/main/java/net/pms/newgui/Splash.java

```diff
@@ -63,6 +61,8 @@ public Splash(PmsConfiguration config) {
 		setLayout(null);
 		add(imglabel);
 		imglabel.addMouseListener(this);
+		img = new ImageIcon(getClass().getResource("/resources/images/icon-32.png"));
+		setIconImage(img.getImage());
```

Comment:
@valib On the "task bar" in Windows for example, or of you use ```Alt + Tab``` to switch foreground application., I'm sure it's used in similar situations for other OS'es.